uml/backend-software-system/screen-handler.puml

```diff
@@ -0,0 +1,51 @@
+@startuml
+
+package "Screen Handler" {
+  class EventType {
+    type : String
+    sprite : Sprite
+  }
+
+  class SpritesReader {
+    +readSprites() : List<Sprite>
+  }
+
+  class ScreenManager {
+    -sprites : List<Sprite>
+    -mapper : ScreenMapper
+    -events : ScreenEvents
+    -settings : ScreenSettings
+    +showSprites(sprites : List<Sprite>)
+  }
+
+  class ScreenMapper {
+    +showSprites(sprites : List<Sprite>)
+    +updateSprite(sprite : Sprite)
+  }
+
+  class ScreenEvents {
+    +listenEvents() : void
+    +onEvent(event : EventType) : void
+  }
+
+  class ScreenSettings {
+    soundVolume : Double
+    musicVolume : Double
+    language : String
```

Comment:
What represents language?

Reply:
As I mentioned before, this class will show user some extra information and language refers to it also. That's why I put it, here is a concept of it.

![image](https://github.com/Pending-Name-21/arquitecture/assets/73414963/50d4ad74-93d5-4cc6-aace-0520e17b73bf)

But what do you think?

Reply:
Yep, it's a nice to have, but the UI it's out of scope.

Reply:
Got it!